cli/internal/specs/v0/source.go

```diff
@@ -89,6 +90,13 @@ func (s *Source) GetWarnings() Warnings {
 	if s.ResourceConcurrency != 0 {
 		warnings["resource_concurrency"] = "the `resource_concurrency` option is deprecated. Please use the plugin-level concurrency option instead"
 	}
+	if s.SkipDependentTables && slices.Contains(s.Tables, "*") {
+		warnings["skip_dependent_tables"] = "the `skip_dependent_tables` option is ineffective when used with '*' `tables`"
+	}
+	if slices.Contains(s.Tables, "*") && len(s.Tables) > 1 {
```

Comment:
Can't we have `tables: ["*", "!aws_ec*"] to exclude some tables?

Reply:
You'd need to add `aws_ec*` to `skip_tables`.